themes/ddbasic/utils.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Utility functions.
+ */
+
+/**
+ * Function for adding classes to body element.
+ */
+function ddbasic_body_class($class = array()) {
+  $classes = &drupal_static(__FUNCTION__, array());
+  if (!empty($class)) {
+    if (!is_array($class)) {
```

Comment:
The $class is not an array we we wrap it in one, so it can be merged with the static $classes array below.
If we combine the if's only strings would be added to the $classes array.